### PR TITLE
signup added and login link corrected

### PR DESCRIPTION
**Pull request by palash23jain**
Screenshot 2022-02-21 at 12 00 00 AM

### CODE REVIEW

File: cart-page/index.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
         </div>
         <input type="text" placeholder="search" class="navbar-search-input">
         <div class="navbar-items">
-            <a href="/index.html" class="btn btn-warning">Login</a>
+            <a href="/login/index.html" class="btn btn-warning">Login</a>
             <a href="/wishlist-page/index.html"><img class="navbar-item" src="/assets/heart.svg" alt=""></a>
             <a href="/cart-page/index.html"><img class="navbar-item" src="/assets/shopping-cart.svg" alt=""></a>
```
Comment (pritam-kr) [inline cart-page/index.html:23]: don't leave alt attribute empty put some relatable text on it.

File: signup/index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            </div>
+            <button class="btn login-btn">Create Account</button>
+            <div class="create-acc">
+                <a href="/login/index.html">Already have an account ></a>
+            </div>
+    </main>
+
+</body>
+
+</html>
```
Comment (pritam-kr) [inline signup/index.html:47]: remove this red mark

File: signup/style.css [hunk 3]
```diff
@@ -0,0 +1,45 @@
+.card-login{
+    max-width: 350px;
```
Comment (pritam-kr) [inline signup/style.css:2]: you can use relative unit after 10px

**Review by pritam-kr — APPROVED** [review]
Good to go. You can merge it

**Review by BelideNikhil — APPROVED** [review]

**Pull request merged**